scripts/integration

```diff
@@ -1,36 +1,16 @@
 #!/bin/bash
 set -ex
 function echo_with_time {
-    echo "$(date --utc +%Y-%m-%dT%H:%M:%SZ) "$@""
+    echo "$(date --utc +%Y-%m-%dT%H:%M:%SZ) $*"
```

Comment:
The `--utc` option isn't recognized on Darwin (`/bin/date` so I'm not running some variant) but `-u` is a shortcut for it on linux. So better to just go with `-u`